src/components/Nav/Nav.scss

```
@@ -0,0 +1,138 @@
+* {
+  list-style-type: none;
+  font-family: 'SUIT', sans-serif;
+  text-decoration: none;
+}
+
+$primary-color: #00388d;
+
+@mixin flex($justify, $align) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+}
+
+.navBox {
+  display: flex;
+  flex-direction: column;
+  align-self: flex-start;
```

flex 위에 잘 선언해주셨는데 못쓰셨군요 🥲

mixin을 추가로 선언하면 어떨까요?

```scss
@mixins columnFlex($jc = center, $ai = center){   <- 이렇게 초기값을 선언해주면 center center정렬일 때는 안써줘도 ㅇㅋ!
  display: flex;
  flex-direction: column;
  justify-content: $jc;
  align-items: $ai;
```

```suggestion
  @include columnFlex(center, flex-start);
```

---

그리고 이부분은 누가 만들어놓지 않았다면! 
styles 폴더에 mixins.scss에 넣어두고 공용으로 사용하면 좋을 것 같아요